PowerEditor/src/NppIO.cpp

```diff
@@ -1900,16 +1907,11 @@ bool Notepad_plus::fileRename(BufferID id)
 		// We are just going to rename the tab nothing else
 		// So just rename the tab and rename the backup file too if applicable
 
-		// https://docs.microsoft.com/en-us/windows/desktop/FileIO/naming-a-file
-		// Reserved characters:  < > : " / \ | ? * tab  
-		//  ("tab" is not in the official list, but it is good to avoid it)
-		std::wstring reservedChars = TEXT("<>:\"/\\|\?*\t");
-
 		std::wstring staticName = _nativeLangSpeaker.getLocalizedStrFromID("tabrename-newname", L"New name");
 
 		StringDlg strDlg;
 		std::wstring title = _nativeLangSpeaker.getLocalizedStrFromID("tabrename-title", L"Rename Current Tab");
-		strDlg.init(_pPublicInterface->getHinst(), _pPublicInterface->getHSelf(), title.c_str(), staticName.c_str(), buf->getFileName(), langNameLenMax - 1, reservedChars.c_str(), true);
+		strDlg.init(_pPublicInterface->getHinst(), _pPublicInterface->getHSelf(), title.c_str(), staticName.c_str(), buf->getFileName(), untitledMaxNameLength, filenameReservedChars.c_str(), true);

```

Comment:
StringDlg class comes from `UserDefineDialog.h`, so please use original value `langNameLenMax - 1` instead of `untitledMaxNameLength`.